src/components/dialogs/filter/expert/expert-filter-form.tsx

```diff
@@ -76,7 +81,12 @@ export const expertFilterSchema = {
 };
 
 function isSupportedEquipmentType(equipmentType: string): boolean {
-    return equipmentType === Generator.type || equipmentType === Load.type;
+    return (
+        equipmentType === Generator.type ||
+        equipmentType === Load.type ||
+        equipmentType === VoltageLevel.type ||
+        equipmentType === Substation.type
+    );
```

Comment:
```suggestion
        return Object.values(EXPERT_FILTER_EQUIPMENTS)
        .map((equipments) => equipments.id)
        .includes(equipmentType);
```
how about future proofing it ?

Reply:
You're right I change it